source/manual/redirect-routes.html.md.erb

```diff
@@ -7,17 +7,131 @@ parent: "/manual.html"
+redirect the old route to another route. This approach is preferred as it
+keeps the workflow and publishing history within the app, as well as
+correctly propagating the changes through Publishing API, Content Store and
+Router API.
+
+Not all applications support unpublishing / setting redirects. In these
+cases, you should change the content item in Publishing API manually.
+
+### Manually apply a redirect in Publishing API
+
+Note that this won't work if the thing you're redirecting originates from
+the Short URL Manager: there's [a rake task][short-url-manager-section-remove-route]
+for that.
+
+You'll need to get the content ID of the current thing that exists at the
+route you want to redirect from. This is generally just a case of visiting
+the route prefixed with `/api/content` to expose its Content Store
+information.
+
+Now, log into the app console:
+
+`$ gds govuk connect app-console -e production publishing_api/publishing-api`
+
+You may want to log into the Content Store app console in another shell:
+
+`$ gds govuk connect app-console -e production content_store/content-store`
+
+In the Content Store console, run
+`content_item = ContentItem.find_by(content_id: "CONTENT_ID")` to get a
+handle on the current state of the content item. If the content item
+already has a `document_type` of `redirect` and the `redirects` property
+has the `path` and `destination` properties you're aiming for, then you
+may just need to [nudge Content Store into talking to Router API][[register-redirect-from-content-store-section].
+
+In the Publishing API console, make a call to `Unpublish`:
+
+```ruby
+Commands::V2::Unpublish.call(
+  content_id: "CONTENT_ID",
+  type: "redirect",
+  explanation: "manually redirected by YOUR_NAME",
+  alternative_path: "/new-path",
+  discard_drafts: true,
+  unpublished_at: Time.now,
+)
+```
+
+You should now see an Unpublishing record in Publishing API:
+
+```ruby
+Unpublishing.last
+=> #<Unpublishing id: 435518, edition_id: 1, type: "redirect", explanation: "manually redirected by Chris", alternative_path: "/new-path", created_at: "2020-11-27 14:03:08", updated_at: "2020-11-27 14:20:47", unpublished_at: nil, redirects: [{"path"=>"/old-path", "type"=>"exact", "destination"=>"/new-path"}]>
+```
+
+And you should see that the content item in the Content Store
+console has been updated, if you run `content_item.reload`.
+
+[Content Store talks directly to Router API](https://github.com/alphagov/content-store/blob/dd79a03d74f130650bc97d1c84aae557ccea58d3/app/models/content_item.rb#L33)
+to update the route. Now when you visit the route in your
+browser, you should be redirected.
+
+### Manually register a route in Content Store
+[register-redirect-from-content-store-section]: #manually-register-a-route-in-content-store
+
+Sometimes Content Store is unsuccessful in updating a route in
+Router API, e.g. if the call from Content Store happens while
+the Router machines are being deployed.
```

Comment:
When Content Store can't set the routes [it will actually remove the content item from its own database](https://github.com/alphagov/content-store/blob/8651b9245046f0a0ae125dde018d39f8507226c5/app/models/content_item.rb#L35), so this might not be the right solution. Instead you'll likely need to [represent it from the Publishing API](https://github.com/alphagov/publishing-api/blob/master/lib/tasks/represent_downstream.rake).

Reply:
It might also be worth mentioning that if Content Store notices the routes on a Content Item haven't changed, [it won't send them to the Router API](https://github.com/alphagov/content-store/blob/8651b9245046f0a0ae125dde018d39f8507226c5/app/models/content_item.rb#L224-L232). This means representing something which has already been published and is in the Content Store but the routes aren't there, won't actually do anything, and the routes _will_ need to registered manually using the command you've given below.

Reply:
Thanks @thomasleese - I've pushed up another commit with a few improvements based on your feedback.